### ISSUE TITLE
Cannot toggle between Editing and Selecting / Tools missing from top toolbar

### ISSUE THREAD

**Issue by freepress529** [WordPress/gutenberg#70281]
### Description

The Tools icon does not exist in the top toolbar. This makes it extremely frustrating to use the Block Editor, as there is no way to toggle between Editing and Selecting.

Wordpress 6.8.1

The Tools icon is documented here, and shown as #3 in the top toolbar: https://wordpress.org/documentation/article/wordpress-block-editor/

### Step-by-step reproduction instructions

Use Block Editor to edit a Page in Wordpress (6.8.1) using the visual editor. The Tools icon does not exist in the top toolbar. There is no apparent way to toggle between Editing and Selecting blocks.

Examining the underlying html code using the (Safari) browser's inspector, Tools is not listed where it should be located between the Block Inserter and Un-do/Re-do items. Tools does not exist.

### Screenshots, screen recording, code snippet

Screenshots and code snippet and additional information shown here:

https://wordpress.org/support/topic/wordpress-block-editor-cannot-toggle-between-editing-and-selecting/

### Environment info

Wordpress 6.8.1 release
Wordpress Twenty Twenty-Five theme
Apple 2021 (Intel) imac, Sequoia 15.5, Safari 18.5, (Firefox also)

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [x] Not sure

**Comment by hamidsafari1996** [comment]
@freepress529 Can you send a picture?

**Comment by freepress529** [comment]
> [@freepress529](https://github.com/freepress529) Can you send a picture?


In the WordPress Block Editor, here is what the top tool bar should be, according to WordPress documentation:

`https://wordpress.org/documentation/article/wordpress-block-editor/`


<img width="1003" alt="Image" src="https://github.com/user-attachments/assets/926a9016-8e37-4179-adb7-ac6fefab5b10" />

<img width="1075" alt="Image" src="https://github.com/user-attachments/assets/ddf8d54f-512c-417c-be4f-10b361d637e7" />



But here is the problem which exists in the WordPress Block Editor.

The Tool menu icon is missing, as indicated where the red arrow is pointing.


<img width="1324" alt="Image" src="https://github.com/user-attachments/assets/efdc779a-11d7-4d08-9d14-c3e46a77e5ef" />



The Safari browser inspector shows the Tools menu icon is also completely missing from underlying html. The Tools menu icon should be listed between the block inserter item and the undo/re-do items, but Tools does not exist there. So, the problem is more than cosmetic, the tool does not exist. This also makes it impossible to switch modes between editing and selecting.

Currently, the Block Editor is very difficult and frustrating to use because this function is missing.



<img width="1711" alt="Image" src="https://github.com/user-attachments/assets/9536fe78-cc2d-4795-82b9-2df7fb077684" />

**Comment by t-hamano** [comment]
Thanks for the report.

> The Tools icon does not exist in the top toolbar.

This is a new intentional behaviour in WordPress 6.8. See #65603. We'd appreciate it if you could comment there about any issues you're experiencing due to the removal of this mode.

I'd like to close this issue as a duplicate of #65603.
